packages/now-build-utils/package.json

```
@@ -16,6 +16,10 @@
     "test-integration-once": "jest --env node --verbose --runInBand test/integration.test.js",
     "prepublishOnly": "./build.sh"
   },
+  "dependencies": {
+    "@now/frameworks": "^0.0.7",
```

```suggestion
    "@now/frameworks": "0.0.7-canary.0",
```

Let use both as canary